minitwit/src/main/java/WebApplication.java

```diff
@@ -556,4 +569,8 @@ else if (!BCrypt.checkpw(enteredPW, rs.getString("pw_hash"))) {
         });
         return filteredMessages.toList();
     };
+
+    public static Route serveSimLatest = (Request request, Response response) -> {
+        return new HashMap<>().put("latest", LATEST);
```

Comment:
This just returns null for me, I think it's because .put returns the value of only the key before changing it. I played around a bit and a solution that worked for me was to store them map in a variable and return that, or change the line to
```
        return Map.ofEntries(
                Map.entry("latest", LATEST)
        );
```

Reply:
Of course! I feel like an idiot. 🤦 Not sure why I thought at the time that it would return the HashMap... I'll change it now.